internal/controller/compute/statefulserverset/statefulserverset.go

```diff
@@ -136,6 +144,12 @@ func (e *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 		e.log.Info("Creating the lans")
 	}
 
+	// ensure that above are already created
+	e.log.Info("Creating the ServerSet CR")
+	if err := e.createSSetCR(ctx, cr); err != nil {
```

Comment:
we should check if the serverset already exists before creating, otherwise we will get an error when `create` is called again if someone adds datavolumes or lans, or if replicas changes. Because we want to create new resources in those cases.